run.py

```diff
@@ -513,28 +514,61 @@ def modifyserviceregs(startval):
                  [_winreg.HKEY_LOCAL_MACHINE,
-                  r'SOFTWARE\Microsoft\Windows\CurrentVersion\DeliveryOptimization\Config',
-                  'DODownloadMode', _winreg.REG_DWORD, defendersenseval],
-
-                 'WifiSense Credential Share': [_winreg.HKEY_LOCAL_MACHINE,
-                                                r'SOFTWARE\Microsoft\WcmSvc\wifinetworkmanager\features',
-                                                'WiFiSenseCredShared', _winreg.REG_DWORD, defendersenseval],
-
-                 'WifiSense Open-ness': [_winreg.HKEY_LOCAL_MACHINE,
-                                         r'SOFTWARE\Microsoft\WcmSvc\wifinetworkmanager\features',
-                                         'WiFiSenseOpen', _winreg.REG_DWORD, defendersenseval],
-
-                 'Windows Defender Spynet': [_winreg.HKEY_LOCAL_MACHINE,
-                                             r'SOFTWARE\Microsoft\Windows Defender\Spynet',
-                                             'SpyNetReporting', _winreg.REG_DWORD, defendersenseval],
-
-                 'Windows Defender Sample Submission': [_winreg.HKEY_LOCAL_MACHINE,
-                                                        r'SOFTWARE\Microsoft\Windows Defender\Spynet',
-                                                        'SubmitSamplesConsent', _winreg.REG_DWORD, defendersenseval]}
+                  r'SOFTWARE\Microsoft\PolicyManager\default\DeliveryOptimization\DODownloadMode',
+                  'value', _winreg.REG_DWORD, defendersenseval],
+                  
+                 'WifiSense Hotspot Auto Connect' :
+                 [_winreg.HKEY_LOCAL_MACHINE,
+                  r'SOFTWARE\Microsoft\PolicyManager\default\WiFi\AllowAutoConnectToWiFiSenseHotspots',
+                  'value', _winreg.REG_DWORD, defendersenseval],
+                  
+                 'WifiSense Hotspot Reporting' :
+                 [_winreg.HKEY_LOCAL_MACHINE,
+                  r'SOFTWARE\Microsoft\PolicyManager\default\WiFi\AllowWiFiHotSpotReporting',
+                  'value', _winreg.REG_DWORD, defendersenseval],
+                  
+                 'Windows Telemtry Data Gathering' :
+                 [_winreg.HKEY_LOCAL_MACHINE,
+                  r'SOFTWARE\Microsoft\PolicyManager\default\System\AllowTelemetry',
+                  'value', _winreg.REG_DWORD, defendersenseval],
+                  
+                 'Windows Defender Automatic Sample Submission' :
+                 [_winreg.HKEY_LOCAL_MACHINE,
+                  r'SOFTWARE\Microsoft\PolicyManager\default\Defender\SubmitSamplesConsent',
+                  'value', _winreg.REG_DWORD, defendersenseval],
+                  
+                 'Windows Defender Spynet (Cloud Protection)' :
+                 [_winreg.HKEY_LOCAL_MACHINE,
+                  r'SOFTWARE\Microsoft\PolicyManager\default\Defender\AllowCloudProtection',
+                  'value', _winreg.REG_DWORD, defendersenseval]}
```

Comment:
Maybe I am missing something here, but is `defendersenseval` really the correct value for all of these? Is it even properly defined in this context since no `global`.